packages/s3-credentials-endpoint/tests/test-index.js

```diff
@@ -342,3 +342,21 @@ test('displayS3Credentials fills template with correct distribution endpoint.',
   await displayS3CredentialInstructions(undefined, res);
   t.true(send.calledWithMatch(expectedLink));
 });
+
+test.serial('An s3credential request with DISABLE_S3_CREDENTIALS set to true results in a 503 error', async (t) => {
+  process.env.DISABLE_S3_CREDENTIALS = true;
+  const username = randomId('username');
+  const accessTokenRecord = fakeAccessTokenFactory({ username });
+  await accessTokenModel.create(accessTokenRecord);
+
+  const response = await request(distributionApp)
+    .get('/s3credentials')
+    .set('Accept', 'application/json')
+    .set('Cookie', [`accessToken=${accessTokenRecord.accessToken}`])
+    .expect(503);
+
+  t.is(response.status, 503);
```

Comment:
Given that the ticket AC mentions that the endpoint when disabled should respond with an informational message, I think we should test that in this unit test. Generally I don't like testing endpoint response messages because its brittle but in this case the ticket seems to call for it.

Reply:
[`e47343b` (#2099)](https://github.com/nasa/cumulus/pull/2099/commits/e47343b5c6d79c1a4e9c81f16100ada84039450d)

Reply:
I **think** it's possible to just check the value of `response.message` and not have to compare against the full JSON response

Reply:
Sorry, I should have been more explicit. You can check `response.body.message` to get the message value that you want to test directly:

https://github.com/nasa/cumulus/blob/master/packages/api/tests/endpoints/test-granules.js#L341

Reply:
[`0ba4cb5` (#2099)](https://github.com/nasa/cumulus/pull/2099/commits/0ba4cb51ffe4fec17799f3419c73f7c327f2824e)